spec/system/jobseekers_can_create_a_job_alert_from_a_search_spec.rb

```diff
@@ -23,7 +23,10 @@
   end
 
   describe "job alert confirmation page" do
+    let(:job_alert_account_creation_prompt_variant) { :bottom }
```

Comment:
you can avoid changing tests by just adding a test block to ab_test.yml with one variant set to 1, if i understand why you have changed these